### PR TITLE
Dev aniket

### PR DESCRIPTION
**Pull request by aniketdevx**

### CODE REVIEW

**Review by udayAlwariya — COMMENTED** [review]
why youve done this

File: index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 </head>
 <body>
     <div>
-        <h1>hellooo</h1>
+        <h1>Aniket 1 hellooo</h1>
```
Comment (udayAlwariya) [inline index.html:10]: why you did this

File: index.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
 </head>
 <body>
     <div>
-        <h1>hellooo</h1>
```
Comment (aniketdevx) [inline index.html:10]: jhdkjdm

File: index.html (hunk 1 above)
Comment (udayAlwariya) [inline index.html:10]: whyyyyyy

File: index.html [hunk 3]
```diff
@@ -7,7 +7,7 @@
 </head>
 <body>
```
Comment (udayAlwariya) [inline index.html:8]: wrrew0rei

**Review by udayAlwariya — APPROVED** [review]

**Pull request merged**